include/dfu/dfu_target_mcuboot.h

```diff
@@ -96,4 +96,8 @@ int dfu_target_mcuboot_done(bool successful);
 
 #endif /* DFU_TARGET_MCUBOOT_H__ */
 
+#ifdef __cplusplus
+}
+#endif
+
```

Comment:
This should be placed before `#endif /* DFU_TARGET_MCUBOOT_H__ */`.

Reply:
I'm impressed I managed to get that wrong :D Thanks